<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecorderDialog</class>
 <widget class="QDialog" name="RecorderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RecorderDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1">
     <item>
      <widget class="QWidget" name="wdg_video" native="true">
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="lb_show">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QGroupBox" name="gb_camera">
           <property name="title">
            <string>摄像头</string>
           </property>
           <widget class="QComboBox" name="cbx_cameraDevc">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>30</y>
              <width>87</width>
              <height>22</height>
             </rect>
            </property>
            <item>
             <property name="text">
              <string>默认设备</string>
             </property>
            </item>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>音频</string>
           </property>
           <widget class="QComboBox" name="cbx_audioDevc">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>30</y>
              <width>87</width>
              <height>22</height>
             </rect>
            </property>
            <item>
             <property name="text">
              <string>默认设备</string>
             </property>
            </item>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="cb_desktop">
           <property name="text">
            <string>桌面</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_camera">
           <property name="text">
            <string>摄像头</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_audio">
           <property name="text">
            <string>音频</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="pb_start">
           <property name="text">
            <string>开始</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_stop">
           <property name="text">
            <string>停止</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lb_url">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>URL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_url">
       <property name="text">
        <string>D:\xuenlei\1.flv</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_setUrl">
       <property name="text">
        <string>设置直播地址</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
